quick-start-with-htap.md

```diff
@@ -0,0 +1,216 @@
+
+This guide walks you through the quickest way to get started with TiDB's one-stop Hybrid Transactional and Analytical Processing (HTAP).
+
+> **Note:**
+>
+> The steps provided in this guide is ONLY for quick start, NOT for production. To explore more features of HTAP, see [explore HTAP](/explore-htap.md).
+
+## Basic concepts
+
+Before using TiDB HTAP, you need to have basic knowledge about [TiKV](/tikv-overview.md), a row-based storage engine for TiDB Online Transactional Processing (OLTP), and [TiFlash](/tiflash/tiflash-overview.md), a columnar storage engine for TiDB Online Analytical Processing (OLAP).
+
+- Storage engines of HTAP: The row-based storage engine and the columnar storage engine co-exist for HTAP. Both storage engines can replicate data automatically and keep strong consistency. The row-based storage engine optimizes OLTP performance, and the columnar storage engine optimizes OLAP performance.
+- Data consistency of HTAP: As a distributed and transactional key-value database, TiKV provides transactional APIs with ACID compliance. With the implementation of the [Raft onsensus algorithm](https://raft.github.io/raft.pdf), TiKV guarantees data consistency between multiple replicas and high availability. TiFlash replicates data from TiKV in real time, which ensures that data is strongly consistent between TiKV.
+- Data isolation of HTAP: TiKV and TiFlash can be deployed on different machines as needed to solve the problem of HTAP resource isolation.
+- MPP computing engine: [MPP](/tiflash/use-tiflash.md#control-whether-to-select-the-mpp-mode) is a distributed computing framework provided by the TiFlash engine since TiDB 5.0, which allows data exchange between nodes and provides high-performance, high-throughput SQL algorithms. If the MPP mode is enabled, the run time of the analytic queries can be significantly reduced.
+
+## Steps
+
+This document take a popular [TPC-H](http://www.tpc.org/tpch/) dataset as an example to guide you to experience the convenience and high performance of TiDB HTAP by trying to query.
+
+### Step 1: Prerequisites for deployment
+
+Before using TiDB HTAP, follow the steps in the [Quick Start Guide for the TiDB Database Platform](/quick-start-with-tidb.md) to deploy a local test environment.
+
+In [Quick Start Guide for the TiDB Database Platform](/quick-start-with-tidb.md):
+
+- You are recommended to run `tiup playground` to start a TiDB cluster of the latest version. When you run the following command, 1 TiDB instance, 1 TiKV instance, 1 PD instance, and 1 TiFlash instance are deloyed automatically:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup playground
+    ```
+
+- If you want to specify the TiDB version and the number of the instances of each component, run the command that specifies the number of the instances of TiFlash as following:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup playground v5.1.0 --db 2 --pd 3 --kv 3 --tiflash 1 --monitor
```

Comment:
- `--monitor` is the default isn't it?
- Shouldn't we use >1 TiFlash to show the MPP benefits?

Reply:
mpp's benefit  need more tiflash node